[#55950] updated documentation for storage health checks

Kharonus opened the pull request:
[#55950](https://community.openproject.org/work_packages/55950)

### WAT?

- updated text and overall structure
- missing current screenshots

### Hint
- @MayaBerd we need to talk about some headings, the future of this page and the screenshots, once you are back.

Kharonus committed:
[#55950] updated documentation for storage health checks

- https://community.openproject.org/work_packages/55950
- updated text and overall structure
- missing current screenshots

dominic-braeunlein commented on docs/system-admin-guide/files/external-file-storages/file-storage-troubleshooting/README.md:
```diff
@@ -8,29 +8,52 @@ keywords: file storages, Nextcloud error, Nextcloud troubleshooting, OneDrive er
+| WRN_UNEXPECTED_CONTENT | The connection request was successful, but unexpected content was found in the drive. | This warning is only shown, if the file storage is configured to automatically manage project folder permissions. There was data found in the drive, that is not a project folder created by OpenProject. | Go to your drive and migrate or delete the data from the drive root, that was no ...
+| ERR_UNKNOWN            | An unknown error occurred.                                                            | There can be multiple reasons and the error source was not foreseen.                                                                                                                                      | Errors of this kind are logged to the server logs. Look for a log entry starting ...
+
+#### Nextcloud compatibility
```
@Kharonus 
I am wondering about the reason of having a own section for "Nextcloud compatibility"?

Kharonus replied:
As just clarified, the heading might be misleading. I tried to have here a placeholder, that explains to the reader, that this is not available for nc file storages (yet). Once the feature is done, this section will be deleted, and the rest of the text updated.

Kharonus committed:
[#55950] updated docs for health status

- added more comprehensible explanation of what the health status is and
  how it is used for troubleshooting
- updated keywords and headlines

Kharonus committed:
[#55950] fixed dead end link

as-op committed:
Merge branch 'refs/heads/release/14.3' into implementation/55950-update-documentation-for-connection-tests

MayaBerd committed:
small update

MayaBerd reviewed (approved):
thanks @Kharonus, i've committed my changes

MayaBerd committed:
Update storages docs 14.3

Kharonus commented:
I am satisfied with the new screenshots and changes by @MayaBerd , tusen takk!

what about you, @dominic-braeunlein ?

dominic-braeunlein reviewed (approved):
LGTM 👍
